ourney/src/main/java/com/lubycon/ourney/domains/trip/entity/Mapping.java

```diff
@@ -0,0 +1,33 @@
+package com.lubycon.ourney.domains.trip.entity;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Getter
+@NoArgsConstructor
```

Comment:
```suggestion
@NoArgsConstructor(access =  AccessLevel.PROTECTED)
```
다른 엔티티에는 다셨는데 요 엔티티에서는 까먹으신 것 같아 서제스천 두림니당